nixos/modules/services/web-apps/mediawiki.nix

```diff
@@ -241,6 +285,12 @@ in
         '';
       };
 
+      webserver = mkOption {
+        type = types.enum [ "apache" "none" ];
```

Comment:
Inconsistent use of `apache` and `httpd` throughout the module.

Reply:
Well what do you prefer?For the nixos options we use httpd. However when specifiing a Webserver name it's httpd

Reply:
What I meant is that you currently doing the following matches:

- 4x `if (cfg.webserver == "httpd")`
- 5x `if (cfg.webserver == "apache")`

The `httpd` case should probably mean `apache` iiuc.